module_intros/locality.Filter.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>Add figure 1 from RAD paper that explains r_ij, r_ik, and theta.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>